tests/docker/BUILD

```diff
@@ -13,10 +13,25 @@
 # limitations under the License.
```

Comment:
How is `tests/docker` different from `tests/container`?

Reply:
tests/docker are tests which run on docker node in bazel ci.
tests/container maps to test for src under container/